### ISSUE TITLE
Ender Shelf text color hard to read on background - when using night vision

### ISSUE THREAD

**Issue by MindCaged** [PrinceOfAmber/Cyclic#1797]
Minecraft Version: 1.16.4

Forge Version: 35.1.37

Mod Version: 1.16.5-1.2.1

Single Player or Server: Single Player

Describe problem (what you were doing; what happened; what should have happened):
I made some Ender Shelves, but the text color is very hard to read on the block's background color. Light Blue on a pinkish background makes it hard to read the names of the enchantments.

Log file link:

Video/images/gifs (direct upload or link): 

![2021-03-26_17 45 25](https://user-images.githubusercontent.com/19498519/112696004-6d309280-8e5b-11eb-8e97-3da58f92b3a5.png)

**Comment by Lothrazar** [comment]
How did you get it to be such a light blue? its really dark for me even in old versions
 
![1797](https://user-images.githubusercontent.com/621529/113637718-25172a00-962a-11eb-9eb7-e6bda95dfe99.png)

**Comment by MindCaged** [comment]
Huh? Must be some sort of mod conflict then? Dang it, I'll just have to live with it, because playing with installed mods until I figure out what's responsible would be a pain in the butt. I suppose I could create a minimalist pack with just Cyclic and maybe JEI and see if it still happens when it's by itself...

Edit: When by itself there's no issue, so some sort of mod conflict with one of the many many mods I have installed.

Edit 2: Okay, I tried loading a different save and the text is black, so something in my save file is making it happen, but who knows what.

Edit 3: I figured it out! I had another mod that gave me semi-permanent night vision effect.
